packages/typo3-version-handling/README.rst

```diff
@@ -0,0 +1,17 @@
+
+==========================================================
+Version handling for PHP based rendering of docs.typo3.org
+==========================================================
+
+This package contains a PHP enum that can be used to determine allowed TYPO3
+versions for documentation interlinking. It also allows a matching between
+descriptive versions like  `stable` and absolute minor versions, for
```

Comment:
Hmm, I don't like the "absolute minor versions" - I just don't understand this term (but I know what you mean).

Reply:
can you do a follow up?